backend/src/main/resources/schema.sql

```diff
@@ -0,0 +1,8 @@
+drop table todo if exists;
```

Comment:
Hibernate is capable to create sql tables from entity. If you use it that way you don't need to create sql files
If you consider to use hibernate way and auto mode use it like this:
spring.jpa.hibernate.ddl-auto=create                    OR
spring.jpa.hibernate.ddl-auto=drop-and-create

Reply:
Done.